pkgs/applications/office/homebank/default.nix

```diff
@@ -17,6 +17,6 @@ stdenv.mkDerivation rec {
     homepage = "http://homebank.free.fr/";
     license = licenses.gpl2Plus;
     maintainers = with maintainers; [ pSub ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Wouldn't it make more sense to have it 

`platforms = [ linux darwin ]`

(untested)

if it'S really running on all macOS devices

Reply:
Nope, `[ linux darwin ]` will create a nested list, since `platforms.linux` and `platforms.darwin` are both lists.